val_scripts/validation-cmssw-benchmarks.sh

```diff
@@ -32,6 +32,12 @@ case ${inputBin} in
         subdir=
         file=pu50-ccc-hs.bin
         ;;
+"112X_TTbar_PU50_MULTI")
```

Comment:
do we get meaningful results with the multi-iter bin files for this non-multi benchmark setup?

Reply:
I don't see why not, but let me test this.

Reply:
is the code in this case selecting iter=0/initialStep or does it inclusively run on all seeds and hits?

Reply:
I believe it uses everything. @osschar should correct me.
In fact, we had talked of replacing CE building with MultiIter building running only for one iteration in the past, but I don't think this was done so far.

Reply:
OK, I will push another commit to do what I described above. Right now, with a partial change:
http://uaf-10.t2.ucsd.edu/~mmasciov/MkFit_pixelPairStep/simval_test_CE_noMultiIter/efficiency/

Efficiency for CE (or STD) is the same as expected (http://uaf-10.t2.ucsd.edu/~mmasciov/MkFit_pixelPairStep/SAV_pixelPairStep/SAV_pixelPairStep_iter4/efficiency/?match=); on the other hand, CMSSW is running over all seeds/hits. Will find a way to solve it, and then everything should be consistent.

Reply:
Yes, I don't think we do any seed filtering in the "old" bh/std/ce building functions. We need to decide how to rewrite them.

Reply:
With the following commits, I'm solving this discrepancy:
http://uaf-10.t2.ucsd.edu/~mmasciov/MkFit_pixelPairStep/SVA_noMultiIter_testV3/
(to be compared, e.g., to http://uaf-10.t2.ucsd.edu/~mmasciov/MkFit_pixelPairStep/SVA_multiIter/simvalseed_test_iter4/, which matches the above multi-iter example, but run on 500 evts as opposed to 1000, for consistency)

Another commit will follow, to fix also other (non-benchmark) validation scripts.

Let me note that builders have been updated to only use initialStep information, for consistency with the past.